!!python/object/new:dynamic_reconfigure.encoding.Config
dictitems:
  flight_time_minute: 5.0
  groups: !!python/object/new:dynamic_reconfigure.encoding.Config
    dictitems:
      flight_time_minute: 5.0
      groups: !!python/object/new:dynamic_reconfigure.encoding.Config
        state: []
      hover_thrust_factor_max: 0.043
      hover_thrust_factor_min: 0.0415
      id: 0
      name: Default
      p_i_acc_error_limit: 0.3
      parameters: !!python/object/new:dynamic_reconfigure.encoding.Config
        state: []
      parent: 0
      position_d_xy: 0.07
      position_d_z: 0.06
      position_ff_xy: 0.6
      position_i_xy: 0.05
      position_i_z: 0.4
      position_p_xy: 2.0
      position_p_z: 3.0
      stand_by_time_minute: 30.0
      state: true
      type: ''
      v_i_acc_error_limit: 0.1
      velocity_d_xy: 0.1
      velocity_d_z: 0.05
      velocity_i_xy: 0.06
      velocity_i_z: 0.04
      velocity_p_xy: 1.5
      velocity_p_z: 1.1
    state: []
  hover_thrust_factor_max: 0.043
  hover_thrust_factor_min: 0.0415
  p_i_acc_error_limit: 0.3
  position_d_xy: 0.07
  position_d_z: 0.06
  position_ff_xy: 0.6
  position_i_xy: 0.05
  position_i_z: 0.4
  position_p_xy: 2.0
  position_p_z: 3.0
  stand_by_time_minute: 30.0
  v_i_acc_error_limit: 0.1
  velocity_d_xy: 0.1
  velocity_d_z: 0.05
  velocity_i_xy: 0.06
  velocity_i_z: 0.04
  velocity_p_xy: 1.5
  velocity_p_z: 1.1
state: []
